src/test/resources/alma/(DE-605)CT003043468.json

```diff
@@ -0,0 +1,126 @@
+{
+  "@context" : "http://lobid.org/resources/context.jsonld",
+  "almaMmsId" : "990197067610206441",
+  "hbzId" : "CT003043468",
+  "responsibilityStatement" : [ "von Erwin Quedenfeldt" ],
+  "title" : "Cöln, Gasthaus \"Zum St. Peter\", Unter Hutmacher 31",
+  "publication" : [ {
+    "location" : "Düsseldorf",
+    "publicationHistory" : "1911",
+    "startDate" : "1911",
+    "type" : [ "PublicationEvent" ],
+    "publishedBy" : "Quedenfeldt"
+  } ],
+  "hasItem" : [ {
+    "heldBy" : {
+      "id" : "http://lobid.org/organisations/DE-605#!",
+      "label" : "lobid Organisation"
+    },
+    "id" : "https://lobid.org/item/990197067610206441",
+    "type" : [ "MBD" ],
+    "label" : "990197067610206441"
+  }, {
+    "id" : "https://lobid.org/item/53723406060006441",
+    "type" : [ "POR" ],
+    "label" : "Portfolio",
+    "heldBy" : {
+      "id" : "http://lobid.org/organisations/DE-61#!",
+      "label" : "lobid Organisation"
+    }
+  }, {
+    "id" : "https://lobid.org/item/53723406060006441",
+    "type" : [ "POR" ],
+    "label" : "Portfolio",
+    "heldBy" : {
+      "id" : "http://lobid.org/organisations/DE-468#!",
+      "label" : "lobid Organisation"
+    }
+  }, {
+    "id" : "https://lobid.org/item/53723406060006441",
+    "type" : [ "POR" ],
+    "label" : "Portfolio"
+  } ],
+  "medium" : [ {
+    "id" : "http://rdaregistry.info/termList/RDAMediaType/1003",
+    "label" : "Datenträger"
+  }, {
+    "id" : "http://rdaregistry.info/termList/RDACarrierType/1018",
+    "label" : "Online-Ressource"
+  } ],
+  "type" : [ "Book", "BibliographicResource" ],
+  "language" : [ {
+    "label" : "Deutsch",
+    "id" : "http://id.loc.gov/vocabulary/iso639-2/ger"
+  } ],
+  "sameAs" : [ {
+    "id" : "http://hub.culturegraph.org/resource/HBZ-CT003043468",
+    "label" : "Culturegraph Ressource"
+  } ],
+  "extent" : "1 Foto ; 24 x 18 cm",
+  "note" : [ "heute Heumarkt 77, \"Gilden im Zims\"" ],
+  "related" : [ {
+    "label" : "Cöln, Gasthaus \"Zum St. Peter\", Unter Hutmacher 31",
+    "id" : "http://lobid.org/resources/HT017551955#!",
+    "note" : "Elektronische Reproduktion von"
+  } ],
+  "inCollection" : [ {
+    "id" : "https://lobid.org/collections#ldd",
+    "type" : [ "Collection" ],
+    "label" : "collections#ldd"
+  }, {
+    "id" : "https://lobid.org/collections#vl-ulbd",
+    "type" : [ "Collection" ],
```

Comment:
The URL does not resolve. Do we need that (new issue)?

Reply:
Could be connected to the inoffical collections or the strange refrences in some elements. Please open a new issue.

Reply:
Opened https://github.com/hbz/lobid-resources/issues/1411.